src/stories/OSCALModification.stories.js

```diff
@@ -10,26 +11,7 @@ const Template = (args) => <OSCALControlModification {...args} />;
 
 export const Default = Template.bind({});
 
-const exampleModifications = {
-  alters: [
-    {
-      "control-id": "control-1",
-      adds: [
-        {
-          position: "starting",
-          props: [
-            {
-              name: "Name",
-              value: "Value",
-            },
-          ],
-        },
-      ],
-    },
-  ],
-};
-
 Default.args = {
-  modifications: exampleModifications,
+  modifications: exampleModificationsAddsTwo,
   controlId: "control-1",
```

Comment:
```suggestion
  controlId: "control-1",
  controlPartId: "control-1_smt.a",
```

Tested this with `npm run storybook` - Without this `controlPartId` prop the icon will be invisible.